common/specification.scs

```diff
@@ -0,0 +1,24 @@
+ps_common_utils_specification
+    <- concept_reusable_component_specification;;
+
+ps_common_utils_specification = [*
+-> rrel_key_sc_element : ps_common_utils
+(*
+    <- concept_reusable_component;;
+    <- concept_atomic_reusable_component;;
+    <- concept_dependent_reusable_component;;
+    <- concept_reusable_ps_component;;
+    <- concept_reusable_dynamically_installed_component;;
+    <- concept_reusable_source_code_component;;
```

Comment:
concept_platform_independent_reusable_component?

Reply:
No, this component is platform dependent

Reply:
yeah, I meant that
but it isn't shown in scs